src/plugins/validation/swagger2/semantic-validators/form-data.js

```diff
@@ -86,18 +87,17 @@ module.exports.validate = function({ resolvedSpec }) {
     );
 
     if (formDataWithTypos.length) {
-      return (errors = errors.concat(
-        params.map((param, i) => {
-          if (param['in'] !== 'formdata') {
-            return;
-          }
-          const pathStr = `${path.join('.')}.${i}`;
-          return {
-            path: pathStr,
-            message: 'The form data value for `in` must be camelCase (formData)'
-          };
-        })
-      ));
+      params.map((param, i) => {
```

Comment:
Since the code is no longer returning anything, `map` is unnecessary. Just use `forEach`
```suggestion
      params.forEach((param, i) => {
```